file_link/hardlink.py

```diff
@@ -0,0 +1,43 @@
+import platform
+
+
+def create_windows(source, destination):
```

Comment:
I would prepend an underscore to highlight that these aren't meant to be used externally.